course_content/notebooks/numpy_intro.ipynb

```diff
@@ -24,7 +24,7 @@
    "source": [
     "## Workshop Objectives\n",
     "\n",
-    "At the end of this workshop you will be able to:\n",
+    "The aim of this workshop is to enable you to:\n",
```

Comment:
This wording is very atypical of a statement of objectives, which is why I changed it. I am not happy with this wording.